### ISSUE TITLE
Demo is broken in current Chrome due to BlobBuilder deprecation/removal

### ISSUE THREAD

**Issue by pauln** [pauln/local-audio-recorder#3]
Current Chrome no longer appears to have a BlobBuilder implementation enabled by default - so the demo does not currently function.  Needs to be reworked to use window.Blob constructor instead.

**Comment by walknick** [comment]
Have you made any progress with this BlobBuilder issue yet? Is there something I can do to encourage you or help you with this? This Flash recorder would really help me move a stalled project forward.

**Comment by pauln** [comment]
Sorry - it was just a matter of getting around to it.  I've just pushed d2fdcaf, which switches the demo over to the Window.Blob() constructor; this means that it should now work in Firefox, Chrome and possibly IE10+ (I don't have a recent enough IE to test with at the moment).

**Comment by walknick** [comment]
Thanks! Really!  I notice the "Uploading, please wait" message doesn't go away after the upload. Is there a way to fix that?

**Comment by pauln** [comment]
Ideally, I'd have built some kind of "reset" function in to the recorder, but I didn't need one and didn't think of it at the time.  I may or may not get around to adding one, but for now 8c28f35 makes it re-initialise the Flash recorder at the end of the callback by removing the object and then replacing the exact same HTML that was removed.

**Comment by walknick** [comment]
Thanks!

**Comment by pauln** [comment]
Quick update on this (for @walknick and anyone else interested): as a result of changes in Flash Player 13 which broke the HTTP POST side of this recorder, I've had to rewrite it to take a separate return URL which the user's browser will be sent to after the upload is complete.  While I was at it, I made it so that a value of "reset" for this parameter will reset the recorder (ready to make another recording) rather than redirecting the browser - so you no longer need to use the remove/replace workaround to reset it.  See the readme for more detail.
